docs/Makefile

```diff
@@ -2,7 +2,7 @@
 #
 
 # You can set these variables from the command line.
-SPHINXOPTS    = -j4
+SPHINXOPTS    = -j4 -W --keep-going
```

Comment:
TIL about the `--keep-going` sphinx option

Reply:
Agreed, I like to see all the warnings and errors in one build log, otherwise you can end up in a very long CI cycle. 😄